core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -203,7 +206,14 @@ else if (element instanceof LikeClause) {
                         throw semanticException(NOT_SUPPORTED, statement, "Only one LIKE clause can specify INCLUDING PROPERTIES");
                     }
                     includingProperties = true;
-                    inheritedProperties = likeTableMetadata.getMetadata().getProperties();
```

Comment:
with this change of behaviour, could you please verify here that also arbitrary / custom property names are being copied (same as before this change) ?